mayastor/src/subsys/opts.rs

```diff
@@ -249,7 +250,7 @@ impl Default for NvmeBdevOpts {
             low_priority_weight: 0,
             medium_priority_weight: 0,
             high_priority_weight: 0,
-            nvme_adminq_poll_period_us: 1_000_000,
+            nvme_adminq_poll_period_us: 100_000,
```

Comment:
Upstream commit c241161 reduced this to 10_000 (10ms).

Reply:
adjusted thx.